### DISCUSSION TITLE
Night Mode

### DISCUSSION THREAD

**Discussion by dragnoxxx** [streamdeck-linux-gui/streamdeck-linux-gui#91] [Ideas / Feature Requests]
Please add a night mode, it litteraly burn my eyes

**Comment by Data-007** [comment]
![2023-10-14_18-03](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/53057579/95388087-b758-4959-bc1c-ac89a9f864de)
 And set brightness in window that pops.
